qiskit_optimization/algorithms/grover_optimizer.py

```diff
@@ -227,6 +226,8 @@ def solve(self, problem: QuadraticProgram) -> OptimizationResult:
                 int_v = self._bin_to_int(v, n_value) + threshold
                 logger.info('Outcome: %s', outcome)
                 logger.info('Value Q(x): %s', int_v)
+                print("rt", rotation_count)
```

Comment:
Remove `print`. If necessary use `logger`.

Reply:
Don't worry. I will remove all the "print"s in the end. I just wanted to see what's happening in the CI.